source/OutfitterPanel.cpp

```diff
@@ -348,7 +359,8 @@ bool OutfitterPanel::CanBuy(bool checkAlreadyOwned) const
 		return false;
 
 	bool isAlreadyOwned = checkAlreadyOwned && IsAlreadyOwned();
-	if(!(outfitter.Has(selectedOutfit) || player.Stock(selectedOutfit) > 0 || isAlreadyOwned))
+	if(!((outfitter.Has(selectedOutfit) && !Imports(customSales, selectedOutfit))
+			|| player.Stock(selectedOutfit) > 0 || isAlreadyOwned))
```

Comment:
Does this mean `import` will prevent an outfitter from selling an outfit it would otherwise be able to sell?

Reply:
Yeah thats the point of import indeed.